data-type/standard/type-Date#exact.yaml

```diff
@@ -0,0 +1,127 @@
+%YAML 1.2
```

Comment:
This file should not exist.  type-Date-exact.yaml is the name of the file that contains this content. The issue is that `type-Date#exact` exists in the GEDCOM v7.1 branch in extracted-files\tags as of August 6th, and type-Date-exact was added Sep 2.  The main branch has only type-Date-exact (since Aug 5).  The fix for this has to be in the GEDCOM repository.